erpnext/manufacturing/doctype/work_order/work_order.py

```diff
@@ -381,13 +382,13 @@ def create_job_card(self):
 				qty = split_qty_based_on_batch_size(self, row, qty)
 				if row.job_card_qty > 0:
 					self.prepare_data_for_job_card(row, index,
-						plan_days, enable_capacity_planning)
+						plan_days, enable_capacity_planning, auto_create_time_log)
 
 		planned_end_date = self.operations and self.operations[-1].planned_end_time
 		if planned_end_date:
 			self.db_set("planned_end_date", planned_end_date)
 
-	def prepare_data_for_job_card(self, row, index, plan_days, enable_capacity_planning):
+	def prepare_data_for_job_card(self, row, index, plan_days, enable_capacity_planning, auto_create_time_log):
```

Comment:
new arguments to existing functions should ideally always be kwarg.

```suggestion
	def prepare_data_for_job_card(self, row, index, plan_days, enable_capacity_planning, auto_create_time_log=False):
```